test/unit/core/bit_manipulation_test.cpp

```diff
@@ -81,29 +83,135 @@ class unsigned_operations : public ::testing::Test
 
 TYPED_TEST_CASE(unsigned_operations, unsigned_types);
 
-TYPED_TEST(unsigned_operations, bit_scan_reverse)
+TYPED_TEST(unsigned_operations, most_significant_bit)
 {
     using unsigned_t = TypeParam;
-    constexpr size_t zero = bit_scan_reverse<unsigned_t>(0b0001);
-    constexpr size_t one1 = bit_scan_reverse<unsigned_t>(0b0010);
-    constexpr size_t one2 = bit_scan_reverse<unsigned_t>(0b0011);
-    constexpr size_t two1 = bit_scan_reverse<unsigned_t>(0b0101);
-    constexpr size_t two2 = bit_scan_reverse<unsigned_t>(0b0111);
-    constexpr size_t seven = bit_scan_reverse<unsigned_t>(0b10010010);
+    constexpr size_t zero  = most_significant_bit<unsigned_t>(0b0001);
+    constexpr size_t one1  = most_significant_bit<unsigned_t>(0b0010);
+    constexpr size_t one2  = most_significant_bit<unsigned_t>(0b0011);
+    constexpr size_t two1  = most_significant_bit<unsigned_t>(0b0101);
+    constexpr size_t two2  = most_significant_bit<unsigned_t>(0b0111);
+    constexpr size_t seven = most_significant_bit<unsigned_t>(0b10010010);
     EXPECT_EQ(zero, 0u);
     EXPECT_EQ(one1, 1u);
     EXPECT_EQ(one2, 1u);
     EXPECT_EQ(two1, 2u);
     EXPECT_EQ(two2, 2u);
     EXPECT_EQ(seven, 7u);
 
-    for (uint8_t position = 0; position < 8u * sizeof(unsigned_t); ++position)
+    for (uint8_t position = 0; position < sizeof_bits<unsigned_t>; ++position)
     {
         unsigned_t start = unsigned_t{1u} << position;
         unsigned_t end = start << 1u;
         for (unsigned_t n = start, k = 0u; n < end && k < max_iterations; ++n, ++k)
         {
-            EXPECT_EQ(bit_scan_reverse(n), position) << "The position of the msb of " << n << " should be " << position;
+            EXPECT_EQ(sdsl::bits::hi(n), position) << "[SDSL] The position of the msb of " << n << " should be "
```

Comment:
why do we test sdsl stuff in SeqAn code?

Reply:
To make sure the SDSL fallback works.
Was requested by @marehr.